src/website/views/components/toolbar/_properties.pug

```diff
@@ -1,6 +1,163 @@
-h2 Web Component
-h3 Properties
-.chi-card.-align--center.-empty
-  .chi-card__content
-    .chi-card__caption
-      | Web component coming soon
+h2 Chi Vue
+h3 Props
+
+h4 Search Input Props
+p.-text
+ | To see props for the <code>search input</code> you must visit its documentation <a href="../forms/search-input/#props">here</a>
+
+h4 Data Table Filters Props
+section.chi-table.chi-table__options.-bordered.-my--3
+  div(style='overflow-x:auto;')
+    table.-text(cellpadding='0' cellspacing='0')
+      thead
+        tr
+          th
+            div Property
+          th
+            div Description
+          th
+            div Type
+          th
+            div Default
+      tbody
+        tr
+          td
+            div
+              code filtersData
+          td
+            div To define filters in the toolbar.
+          td
+            div
+              code DataTableFilters[]
+          td
+            div
+              code []
+        tr
+          td
+            div
+              code portal
+          td
+            div To define portal style of the toolbar.
+          td
+            div
+              code DataTableFilters[]
+          td
+            div
+              code []
```

Comment:
```suggestion
```
Please remove portal props as it is not neccesary anymore

Reply:
Solved.